app/pages/clinicworkspace/ClinicPatients.js

```diff
@@ -455,7 +455,7 @@ export const ClinicPatients = (props) => {
   const [showClinicPatientTagsDialog, setShowClinicPatientTagsDialog] = useState(false);
   const [showTimeInRangeDialog, setShowTimeInRangeDialog] = useState(false);
   const [showSendUploadReminderDialog, setShowSendUploadReminderDialog] = useState(false);
-  const [showNames, setShowNames] = useState(false);
+  const [showNames, setShowNames] = useState(true);
```

Comment:
Pending Investigation: Ticket story mentions clinicians but not sure if the intended behaviour it also applies to Clinics. Will need clarification from product/design.

Reply:
clarified: default view is for BOTH clinicians and clinic admins